spec/models/company_spec.rb

```diff
@@ -3,6 +3,7 @@
 RSpec.describe Company, type: :model do
   let(:company) { build(:company) }
 
+  # FIXME: Maybe? I don't see a class method by this name
```

Comment:
This is defined in `app/models/concerns/queryable.rb`. I would have liked to have tested the `.get_record_val_by` method inside of the spec file for the `Queryable` module, but then I'd have to start stubbing an entire database table, and I'm not sure how to do that. Instead, I test that method from the spec file for models which include `Queryable`